pycodestyle.py

```diff
@@ -56,6 +56,15 @@
 import time
 import tokenize
 import warnings
+try:
+    from functools import lru_cache
+except ImportError:
+    def lru_cache(maxsize=128):  # noqa as it's a fake implementation.
```

Comment:
Sorry, I misspoke. The minimum supported Python 3 version is 3.3. I'm not sure if @IanLee1521 wants to continue to support Python 2.6.